### ISSUE TITLE
Depends: pulseaudio-module-gconf

### ISSUE THREAD

**Issue by ratsalad** [feren-OS/Feren-OS-Dependency-Issue-Reporting#11]
**Make sure you have the default repository set in Feren OS**
If you have any other repositories enabled in Feren OS outside of the default set of repositories, then this might be the cause of your package's dependency issue.

**What is the name of the troublesome package?**
Please put the name of the package here:Depends: pulseaudio-module-gconf

**What is the output of an apt-get install attempt for this package?**
Please copy and paste the output of apt-get install (troublesome package name here) below in the space between the two triple-`s:
```
Checking for the HWE Stack and if it's not installed installing it

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
linux-generic-hwe-18.04 is already the newest version (5.0.0.37.95).
xserver-xorg-hwe-18.04 is already the newest version (1:7.7+19ubuntu8~18.04.3).
The following packages were automatically installed and are no longer required:
  feren-backgrounds-xenon feren-backgrounds-yttrium libasound2-dev
  libcapnp-0.6.1 libexiv2-27 libgles2-mesa-dev libmirclient-dev libmirclient9
  libmircommon-dev libmircommon7 libmircookie-dev libmircookie2 libmircore-dev
  libmircore1 libmirprotobuf3 libprotobuf-dev libsndio-dev libvte-common
  libvte9 libxcursor-dev libxi-dev libxinerama-dev libxrandr-dev
  libxrender-dev libxss-dev libxv-dev python-pycurl python-vte
  x11proto-randr-dev x11proto-scrnsaver-dev x11proto-xinerama-dev
Use 'apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.





Installing the new Feren OS experience... This WILL take a while...

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
xserver-xorg-input-wacom-hwe-18.04 is already the newest version (1:0.36.1-0ubuntu1~18.04.1).
The following packages were automatically installed and are no longer required:
  feren-backgrounds-xenon feren-backgrounds-yttrium libasound2-dev
  libcapnp-0.6.1 libexiv2-27 libgles2-mesa-dev libmirclient-dev libmirclient9
  libmircommon-dev libmircommon7 libmircookie-dev libmircookie2 libmircore-dev
  libmircore1 libmirprotobuf3 libprotobuf-dev libsndio-dev libvte-common
  libvte9 libxcursor-dev libxi-dev libxinerama-dev libxrandr-dev
  libxrender-dev libxss-dev libxv-dev python-pycurl python-vte
  x11proto-randr-dev x11proto-scrnsaver-dev x11proto-xinerama-dev
Use 'apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
Package 'feren-oobe' is not installed, so not removed
The following packages were automatically installed and are no longer required:
  feren-backgrounds-xenon feren-backgrounds-yttrium libasound2-dev
  libcapnp-0.6.1 libexiv2-27 libgles2-mesa-dev libmirclient-dev libmirclient9
  libmircommon-dev libmircommon7 libmircookie-dev libmircookie2 libmircore-dev
  libmircore1 libmirprotobuf3 libprotobuf-dev libsndio-dev libvte-common
  libvte9 libxcursor-dev libxi-dev libxinerama-dev libxrandr-dev
  libxrender-dev libxss-dev libxv-dev python-pycurl python-vte
  x11proto-randr-dev x11proto-scrnsaver-dev x11proto-xinerama-dev
Use 'apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
gnome-themes-standard is already the newest version (3.28-1ubuntu1).
policykit-desktop-privileges is already the newest version (0.20ubuntu18.04.1).
kdelibs-bin is already the newest version (4:4.14.38-0ubuntu3.1).
kdelibs-bin set to manually installed.
kdelibs5-data is already the newest version (4:4.14.38-0ubuntu3.1).
kdelibs5-data set to manually installed.
kdelibs5-plugins is already the newest version (4:4.14.38-0ubuntu3.1).
kdelibs5-plugins set to manually installed.
kio is already the newest version (5.65.0-0xneon+18.04+bionic+build48).
phonon-backend-gstreamer is already the newest version (4:4.10.0-0xneon+18.04+bionic+build10).
phonon-backend-gstreamer set to manually installed.
onboard is already the newest version (1.4.1-2ubuntu1mint2).
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 plasma-pa : Depends: pulseaudio-module-gconf but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
Reading package lists...
Building dependency tree...
Reading state information...
The following packages were automatically installed and are no longer required:
  feren-backgrounds-xenon feren-backgrounds-yttrium libasound2-dev
  libcapnp-0.6.1 libexiv2-27 libgles2-mesa-dev libmirclient-dev libmirclient9
  libmircommon-dev libmircommon7 libmircookie-dev libmircookie2 libmircore-dev
  libmircore1 libmirprotobuf3 libprotobuf-dev libsndio-dev libvte-common
  libvte9 libxcursor-dev libxi-dev libxinerama-dev libxrandr-dev
  libxrender-dev libxss-dev libxv-dev python-pycurl python-vte
  x11proto-randr-dev x11proto-scrnsaver-dev x11proto-xinerama-dev
Use 'apt autoremove' to remove them.
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
gnome-themes-standard is already the newest version (3.28-1ubuntu1).
policykit-desktop-privileges is already the newest version (0.20ubuntu18.04.1).
kdelibs-bin is already the newest version (4:4.14.38-0ubuntu3.1).
kdelibs-bin set to manually installed.
kdelibs5-data is already the newest version (4:4.14.38-0ubuntu3.1).
kdelibs5-data set to manually installed.
kdelibs5-plugins is already the newest version (4:4.14.38-0ubuntu3.1).
kdelibs5-plugins set to manually installed.
kio is already the newest version (5.65.0-0xneon+18.04+bionic+build48).
phonon-backend-gstreamer is already the newest version (4:4.10.0-0xneon+18.04+bionic+build10).
phonon-backend-gstreamer set to manually installed.
onboard is already the newest version (1.4.1-2ubuntu1mint2).
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 plasma-pa : Depends: pulseaudio-module-gconf but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
Gtk-Message: 10:45:30.266: GtkDialog mapped without a transient parent. This is discouraged.


```

Thank you for helping me fix as many dependency issues as possible. When you're ready to report, click 'Submit new issue'. You can be contacted by The Developer or others via replies to this issue in the future, so please be sure to check GitHub Notifications during the time this issue is open, otherwise this issue could be closed due to inactivity from those suffering from this dependency issue.

**Comment by dominichayesferen** [comment]
Try going into Software Sources > Maintenance > Downgrade Foreign Packages, and if there are any pulseaudio packages listed downgrade them.

Once that is done, try the transition again.
